examples/arithmetics/src/language-server/arithmetics-util.ts

```diff
@@ -4,9 +4,11 @@
  * terms of the MIT License, which is available in the project root.
  ******************************************************************************/
 
-export function applyOp(op: '+' | '-' | '*' | '/'): (x: number, y: number) => number {
+export function applyOp(op: '+' | '-' | '*' | '/' | '^' | '%'): (x: number, y: number) => number {
```

Comment:
```suggestion
export function applyOp(op: BinaryExpression['operator']): (x: number, y: number) => number {
```

Reply:
Same thoughts apply here

Reply:
This would be quite nice to add actually. No need to re-declare the binary ops.